configure.sh

```diff
@@ -5,6 +5,49 @@ export DEBIAN_FRONTEND=noninteractive
 echo "US/Central" | sudo tee /etc/timezone
 dpkg-reconfigure --frontend noninteractive tzdata
 
+# Let's read some configuration information
+# from the user.
+echo -n "Reading configuration..."
+source /home/vagrant/configuration/configuration.env > /dev/null 2>&1
+if [ $? -ne 0 ];
+then
+	echo "failure. Could not source your configuration/configuration.env file."
+	exit 1
+fi
+echo "done."
+
+# Now, check to make sure that all relevant variables are configured
+# with sane values.
+if [ "x${OTS_SVN_USERNAME}" == "x" ];
```

Comment:
Is there a reason you didn't use `-z` as in

```
if [ -z "${OTS_SVN_USERNAME}" ];
```

It's a bit clearer than this method.

Reply:
Just a bit of my heritage -- the "x" trick I picked up from autoconf configure scripts. Happy to make this change, too.